### PR TITLE
Update Server Console, Address Overflows

### PR DESCRIPTION
**Pull request by notAreYouScared**
After, Dark: image

After, Light ( Still needs some love ):
image

### CODE REVIEW

File: app/Filament/Server/Widgets/ServerConsole.php [hunk 1]
```diff
@@ -57,6 +57,11 @@ protected function getSocket(): string
         return $socket;
     }
 
+    protected function canSendCommand(): bool
+    {
+        return !($this->server->isInConflictState() || $this->server->retrieveStatus() == 'offline');
```
Comment (Boy132) [inline app/Filament/Server/Widgets/ServerConsole.php:62]:
```suggestion
        return !$this->server->isInConflictState() && $this->server->retrieveStatus() === 'running';
```

File: resources/views/filament/components/server-console.blade.php [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
 
-    <div>
+    <div class="flex items-center w-full bg-transparent border rounded">
+        <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
+             stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"
+             class="icon icon-tabler icons-tabler-outline icon-tabler-chevrons-right">
+            <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+            <path d="M7 7l5 5l-5 5" />
+            <path d="M13 7l5 5l-5 5" />
+        </svg>
```
Comment (Boy132) [inline resources/views/filament/components/server-console.blade.php:33]: Is there a reason you don't use `<x-filament-icon>` (or `@svg`)?

**Review by Boy132 — CHANGES_REQUESTED** [review]

File: resources/views/filament/components/server-console.blade.php [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
-            integrity="sha512-Gujw5GajF5is3nMoGv9X+tCMqePLL/60qvAv1LofUZTV9jK8ENbM9L+maGmOsNzuZaiuyc/fpph1KT9uR5w3CQ=="
-            crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/xterm/5.5.0/xterm.css"
-          integrity="sha512-AbNrj/oSHJaILgcdnkYm+DQ08SqVbZ8jlkJbFyyS1WDcAaXAcAfxJnCH69el7oVgTwVwyA5u5T+RdFyUykrV3Q=="
-          crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/xterm/css/xterm.css">
+    <script src="https://cdn.jsdelivr.net/npm/xterm/lib/xterm.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/xterm-addon-fit/lib/xterm-addon-fit.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/xterm-addon-web-links/lib/xterm-addon-web-links.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/xterm-addon-search/lib/xterm-addon-search.js"></script>
```
Comment (Boy132) [inline resources/views/filament/components/server-console.blade.php:7]: These npm packages are deprecated apparently.

```suggestion
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@xterm/xterm/css/xterm.css">
    <script src="https://cdn.jsdelivr.net/npm/@xterm/xterm/lib/xterm.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@xterm/addon-fit/lib/xterm-addon-fit.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@xterm/addon-web-links/lib/xterm-addon-web-links.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@xterm/addon-search/lib/xterm-addon-search.js"></script>
```

**Review by Boy132 — CHANGES_REQUESTED** [review]

File: resources/views/filament/components/server-console.blade.php (hunk 3 above)
Comment (notAreYouScared) [reply resources/views/filament/components/server-console.blade.php:7]: I see, I was just going off the old console area.

File: resources/views/filament/components/server-console.blade.php (hunk 2 above)
Comment (notAreYouScared) [reply resources/views/filament/components/server-console.blade.php]: If i knew it was a thing i would've used it <3

**Review by lancepioch — APPROVED** [review]

**Review by Boy132 — APPROVED** [review]

**Pull request merged**